.github/workflows/ci.yml

```diff
@@ -17,7 +17,9 @@ jobs:
           images: redpwn/jail
           tags: |
             type=sha,format=long
-            type=ref,event=tag
+            type=semver,pattern={{version}}
+            type=semver,pattern={{major}}.{{minor}}
+            type=semver,pattern={{major}}
```

Comment:
This drops the `v` prefix to tags. Since we only ever published a full-version tag previously (not `major` or `major.minor`), this can't break anybody. Is this intentional? Should we document semver compliance + the tags we offer?